test-cases/tests/multiAZ/p01-measure-downtime-during-down-az.md

```diff
@@ -45,71 +46,135 @@ services remain available.
    make local/deploy
    ```
 
-   See step 8 and 9, you might want to do these before killing the zone as well.
-
-4. In terminal window #2, run the following [script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/disableAz.sh)
-   to fail an AZ
-
-   ```bash
-   # check the az of the current cluster e.g.
-   oc get machineset -n openshift-machine-api
-   NAME                                      DESIRED   CURRENT   READY   AVAILABLE   AGE
-   mw-collab-multi-45qpn-infra-eu-west-1a    1         1         1       1           70m
-   mw-collab-multi-45qpn-infra-eu-west-1b    1         1         1       1           70m
-   mw-collab-multi-45qpn-infra-eu-west-1c    1         1         1       1           70m
-   mw-collab-multi-45qpn-worker-eu-west-1a   3         3         3       3           94m
-   mw-collab-multi-45qpn-worker-eu-west-1b   3         3         3       3           94m
-   mw-collab-multi-45qpn-worker-eu-west-1c   3         3         3       3           94m
-   # run the script to disable the AZ e.g.
-   ./scripts/disableAz.sh true eu-west-1a
-   ```
-
-5. Wait around 30 minutes for all terminating pod to redeploy to their new AZ's. You can use the following
-   [script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/podsAz.sh) to check the pod
-   distribution.
+4. Record the pod distribution for 3scale, user SSO, cluster SSO, and marin3r using this [script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/podsAz.sh)
```

Comment:
might want to include the middleware-monitoring

Reply:
Good point, updated. Also, I changed the namespace names so that they matches the real ones (without namespace prefix)